### PR TITLE
AzureRiskyUsers v1.1.36 - azure-risky-users-list extension

### PR DESCRIPTION
**Pull request by mohlcyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
- Added the *order_by* argument to ***azure-risky-users-list*** command.
- Added the *updated_before* argument to ***azure-risky-users-list*** command.
- Added the *updated_after* argument to ***azure-risky-users-list*** command.
- Updated the Docker image to: *demisto/crypto:1.0.0.96042*.

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.py [hunk 1]
```diff
@@ -96,16 +96,21 @@ def get_token_retrieval_url_by_auth_type(authentication_type: str) -> None | str
...
+                                 update_before: str | None = None,
+                                 updated_after: str | None = None) -> dict:
         """
         List risky users.
 
         Args:
             risk_state (str): Risk State to retrieve.
             risk_level (str): Specify to get only results with the same Risk Level.
             limit (int): Limit of results to retrieve.
+            order_by (str): Order results by this attribute.
```
Comment (RotemAmit) [inline Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.py:113]: Please add the rest of the arguments to the doc string as well (update_before, updated_after). Here and in the rest of the functions.

**Review by RotemAmit — COMMENTED** [review]
Thank you for your contribution! Nice work!
Feel free to reach out if you have any question here or on Slack.
Some general notes:
* Please fix the errors in the pre-commit step ([link](https://github.com/demisto/content/actions/runs/9309590392/job/25625326218?pr=34654)).
* Add unit tests to tests your updates. You can find more information about writing unit tests [here](https://xsoar.pan.dev/docs/integrations/unit-testing).

File: Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.py (hunk 1 above)
Comment (mohlcyber) [reply Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.py:113]: Done updated the integration accordingly.

**Review by RotemAmit — COMMENTED** [review]
Great work! Let's schedule a quick demo.
I asked the tech-doc team to review the documentation.

File: Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.yml [hunk 2]
```diff
@@ -82,6 +82,17 @@ script:
...
       - medium
       - high
+    - description: The method used to order the retrieved results.
+      name: order_by
+      defaultValue: 'riskLastUpdatedDateTime desc'
+      auto: PREDEFINED
+      predefined:
+        - 'riskLastUpdatedDateTime desc'
+        - 'riskLastUpdatedDateTime asc'
+    - description: Displays all RiskyUsers before a specific Date Time. For Example "2024-02-27T04:49:26.257525Z", "10 days", "5 months", "2 hours".
```
Comment (ShirleyDenkberg) [inline Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.yml:92]:
```suggestion
    - description: Displays all RiskyUsers before a specific datetime. For Example "2024-02-27T04:49:26.257525Z", "10 days", "5 months", "2 hours".
```

File: Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.yml [hunk 3]
```diff
@@ -82,6 +82,17 @@ script:
...
+      name: updated_before
+    - description: Displays all RiskyUsers after a specific Date Time. For Example "2024-02-27T04:49:26.257525Z", "10 days", "5 months", "2 hours".
```
Comment (ShirleyDenkberg) [inline Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.yml:94]:
```suggestion
    - description: Displays all RiskyUsers after a specific datetime. For Example "2024-02-27T04:49:26.257525Z", "10 days", "5 months", "2 hours".
```

File: Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/README.md [hunk 4]
```diff
@@ -181,9 +181,13 @@ Returns a list of all risky users and their properties.
 | --- | --- | --- |
 | risk_state | Sets the Risk State to retrieve. Possible values are: atRisk, confirmedCompromised, remediated, dismissed. | Optional | 
 | limit | Limit of results to retrieve. Default is 50. | Optional | 
+| page | Page number. | Optional | 
 | page_size | Amount of results per request. Value can be between 1 and 500. When only page_size is given, the first page results will be fetched. | Optional | 
 | next_token | The URL for the next set of items to return during pagination. (This URL can be retrieved from a previous call). | Optional | 
 | risk_level | Sets the Risk Level to retrieve. Possible values are: low, medium, high. | Optional | 
+| order_by | The method used to order the retrieved results. Possible values are: riskLastUpdatedDateTime desc, riskLastUpdatedDateTime asc. Default is riskLastUpdatedDateTime desc. | Optional | 
+| updated_before | Displays all RiskyUsers before a specific Date Time. For Example "2024-02-27T04:49:26.257525Z", "10 days", "5 months", "2 hours". | Optional | 
+| updated_after | Displays all RiskyUsers after a specific Date Time. For Example "2024-02-27T04:49:26.257525Z", "10 days", "5 months", "2 hours". | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/README.md:190]:
```suggestion
| updated_before | Displays all RiskyUsers before a specific datetime. For Example "2024-02-27T04:49:26.257525Z", "10 days", "5 months", "2 hours". | Optional | 
| updated_after | Displays all RiskyUsers after a specific datetime. For Example "2024-02-27T04:49:26.257525Z", "10 days", "5 months", "2 hours". | Optional | 
```

File: Packs/AzureRiskyUsers/ReleaseNotes/1_1_37.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Azure Risky Users
+
+- Added 3 new arguments to the command ***azure-risky-users-list***:
```
Comment (ShirleyDenkberg) [inline Packs/AzureRiskyUsers/ReleaseNotes/1_1_37.md:6]:
```suggestion
- Added 3 new arguments to the ***azure-risky-users-list*** command:
```

**Comment by mohlcyber** [comment]
Hi @RotemAmit, 

the recording is posted here: https://drive.google.com/file/d/1wdsKiMW_TIS9NiMBFnTnqDGNR2dkFVnV/view?usp=sharing
If you need any additional details please let me know.

Many thanks

**Review by RotemAmit — APPROVED** [review]
Thank you very much for your contribution! The demo looks great!

**Pull request merged**